### PR TITLE
many: simplify release package and add OnClassic

### PR DESCRIPTION
**Pull request by niemeyer**

### CODE REVIEW

File: release/release.go [hunk 1]
```diff
@@ -108,3 +64,26 @@ func ReadLsb() (*Lsb, error) {
 
 	return lsb, nil
 }
+
+var dpkgStatusPath = "/var/lib/dpkg/status"
+
+// OnClassic returns whether the process is running inside a
+// classic Ubuntu system or a native Ubuntu Core image.
+func OnClassic() bool {
```
Comment (pedronis) [inline release/release.go]: this could use osutil.FileExists, it's exactly this pattern of using stat

Comment (pedronis) [inline release/release.go]: also wondering if setting/caching this in a init function would make sense

Comment (niemeyer) [inline release/release.go]: Fixed, thanks. I considered caching, but decided it wasn't worth the trouble for now. We'll need to consider concurrency and will need to reset for tests, which is easy but more complex. If we end up using this in a relevant path that justifies the complexity, happy to fix it.

Comment (niemeyer) [inline release/release.go]: Actually, I take that back. It's actually even simpler if we cache it. Let me give it a try.

**Comment by mvo5** [comment]
:+1:

**Comment by pedronis** [comment]
LGTM

**Comment by mvo5** [comment]
retest this please

**Comment by pedronis** [comment]
needs go fmt

**Comment by pedronis** [comment]
picked up in #1132 to fix the integration test failure

**Pull request closed without merging**